django_object_actions/templates/django_object_actions/action_trigger.html

```diff
@@ -0,0 +1,22 @@
+{% load add_preserved_filters from admin_urls %}
+
+{% if tool.button_type == 'a' %}
+  <a href="{% add_preserved_filters action_url %}"
+    title="{{ tool.standard_attrs.title }}"
+    {% for k, v in tool.custom_attrs.items %}
+      {{ k }}="{{ v }}"
+    {% endfor %}
+    class="{{ tool.standard_attrs.class }}"
+  >{{ tool.label|capfirst }}</a>
+{% elif tool.button_type == 'form' %}
+  <form method="post" action="{% add_preserved_filters action_url %}">
+    {% csrf_token %}
+    <a href="#" onclick="this.parentNode.submit(); return false;"
```

Comment:
I really doubt Django has styling for anything other than the `<a>` but it would be nice to use `<button`> or `<input type=submit>` and not need the JavaScript.